data/templates/openvpn/server.conf.tmpl

```diff
@@ -127,6 +127,16 @@ push "dhcp-option DNS6 {{ nameserver }}"
 {%     if server.domain_name is defined and server.domain_name is not none %}
 push "dhcp-option DOMAIN {{ server.domain_name }}"
 {%     endif %}
+{%     if server['2fa'] is defined and server['2fa'] is not none %}
+{%       if server['2fa']['totp'] is defined and server['2fa']['totp'] is not none %}
+plugin "/usr/lib/openvpn/openvpn-otp.so" "otp_secrets=/config/auth/openvpn/{{ ifname }}-otp-secrets otp_slop=
```

Comment:
Maybe define the plugin dir as a constant somewhere?

Reply:
Please use the common Jinja2 style of `if server.2fa is defined and server.2fa is not none`